static/app/components/avatarChooser/index.tsx

```diff
@@ -398,17 +400,13 @@ const CropperHovercard = styled(Hovercard)`
   width: 300px;
 `;
 
-const CropperContainer = styled('div')`
-  display: flex;
-  flex-direction: column;
-  gap: ${space(2)};
-`;
+function CropperContainer(props: FlexProps) {
+  return <Stack gap="xl" {...props} />;
+}
```

Comment:
this only has one usage, so I think we should inline it. In fact, I think we should inline most things. The name `CropperContainer` doesn’t tell us anything and I always wind up looking at the implementation to get what it does :/